templates/commands/golden_test/test_record.go

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package test implements the template test related subcommands.
-package test
+// Package golden_test implements golden test related subcommands.
+package golden_test
```

Comment:
By convention, package names don't contain underscores.

Reply:
Got that! updated to goldentest, same for folder name.